### PR TITLE
✨Source Pivotal-Tracker: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #36498: ✨Source Pivotal-Tracker: Migrate Python CDK to Low-code CDK
### Topic

Migrate Pivotal-Tracker Source to Low-code CDK

### Relevant information

https://docs.airbyte.com/integrations/sources/pivotal-tracker

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Pivotal Tracker to Low-Code CDK*
Closes airbytehq/airbyte-internal-issues#7010

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>

 - Full passed state:
 
image

</details>

## 🚨 User Impact 🚨
- Spec Breaking changes, migration to low-code

### CODE REVIEW

**Comment by btkcodedev** [comment]
Built using connector builder,
airbyte-ci test -> Full passed state

File: airbyte-integrations/connectors/source-pivotal-tracker/metadata.yaml [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
-  remoteRegistries:
-    pypi:
-      enabled: true
-      packageName: airbyte-source-pivotal-tracker
-  registries:
-    cloud:
-      enabled: false
-    oss:
-      enabled: true
+  releaseDate: 2024-04-01
```
Comment (girarda) [inline airbyte-integrations/connectors/source-pivotal-tracker/metadata.yaml:28]:
```suggestion
  releaseDate: 2022-04-04
```

**Comment by btkcodedev** [comment]
Inline Schema ✅
Poetry lock file ✅
No Breaking change ✅
Docs updated ✅

`airbyte-ci test` All pass state:
Results:
image

**Comment by btkcodedev** [comment]
Inline Schema ✅
Poetry lock file ✅
No Breaking change ✅
Docs updated ✅

`airbyte-ci test` All pass state:
Results:
image

**Pull request closed without merging**